javascript/javascript1/week1/dog_age_calculator.js

```diff
@@ -0,0 +1,11 @@
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2030;
+let dogYear = (dogYearFuture - dogYearOfBirth); 
+let shouldShowResultInDogYears = true;
+
+
+if (shouldShowResultInDogYears === true) {
+    console.log(`Your dog will be ${dogYear} human years old in ${dogYearFuture}`)
+} else {
+    console.log(`Your dog will be ${dogYear * 7} human years old in ${dogYearFuture}`)
+}
```

Comment:
You can simply use `const` for those variables which you are sure you will not override in the future of your code.

Reply:
I have updated variable from let to const